zenodo/modules/stats/registrations.py

```diff
@@ -121,3 +122,102 @@ def register_aggregations():
             )),
 
     ]
+
+
+def register_queries():
+    """Register Zenodo queries."""
+    return [
+        dict(
+            query_name='record-download',
+            query_class=ESTermsQuery,
+            query_config=dict(
+                index='stats-file-download',
+                doc_type='file-download-day-aggregation',
+                copy_fields=dict(
+                    bucket_id='bucket_id',
+                    record_id='record_id',
+                    recid='recid',
+                    conceptrecid='conceptrecid',
+                    doi='doi',
+                    conceptdoi='conceptdoi',
+                    communities=lambda d, _: (list(d.communities)
+                                              if d.communities else None),
+                    is_parent=lambda *_: False
+                ),
+                required_filters=dict(
+                    recid='recid',
+                ),
+                metric_fields=dict(
+                    unique_count=('sum', 'unique_count'),
+                    volume=('sum', 'volume'),
+                )
+            ),
+        ),
+        dict(
+            query_name='record-download-all-versions',
+            query_class=ESTermsQuery,
+            query_config=dict(
+                index='stats-file-download',
+                doc_type='file-download-day-aggregation',
+                copy_fields=dict(
+                    conceptrecid='conceptrecid',
+                    conceptdoi='conceptdoi',
+                    communities=lambda d, _: (list(d.communities)
+                                              if d.communities else None),
+                    is_parent=lambda *_: True
```

Comment:
Hm, we need top implement a `default_filter` on the `*-all-versions` queries where we can specify `is_parent:true`